src/main/java/Bebe.java

```diff
@@ -0,0 +1,149 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+
+public class Bebe {
+
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void main(String[] args) {
+        welcomeMessage();
+        runChatbot();
+        exitMessage();
+    }
```

Comment:
The main is very clear and concise, enhancing the readability and maintainability.